src/libcore/pin.rs

```diff
@@ -170,7 +329,12 @@ where
     P::Target: Unpin,
 {
     /// Construct a new `Pin` around a pointer to some data of a type that
```

Comment:
```suggestion
    /// Construct a new `Pin<P>` around a pointer to some data of a type that
```

Reply:
Why remove the space?

Reply:
WTF, it's 4 spaces both before and after the patch but GH shows a diff?!?